src/components/card/LetterCard.jsx

```
@@ -8,9 +11,26 @@ const LetterCard = ({
+  const { textColor } = useSelector((state) => state.theme);
   return (
-    <div onClick={onClick} className={` ${!signImage && "pb-5"} letter-card`}>
+    <div
+      onClick={onClick}
+      className={` ${!signImage && "pb-5"} letter-card relative`}
+    >
+      {for_modify && (
+        <div
+          className="absolute top-0 right-1 flex gap-2 z-10"
+          onClick={(e) => e.stopPropagation()} // Prevent click bubbling to main card
+        >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)